app/components/BottomArcsiPlayer.vue

```diff
@@ -31,6 +31,10 @@
 import { mediaServerURL } from '~/constants'
 
 export default {
+  validate ({ params, store }) {
+    // Check if arcsiShows exists
```

Comment:
Where will this `validate()` method be called?

Reply:
https://nuxtjs.org/docs/components-glossary/validate/#the-validate-method